app/locale/en_US/Oyst_OneClick.csv

```diff
@@ -136,4 +136,6 @@
 "Click <a href=""%s"">here</a> to set up your settings.","Click <a href=""%s"">here</a> to set up your settings."
 "Resetting the module","Resetting the module"
 "OneClick module was reinstalled successfully.","OneClick module was reinstalled successfully."
-"OneClick module was unable to reinstall successfully.","OneClick module was unable to reinstall successfully."
\ No newline at end of file
+"OneClick module was unable to reinstall successfully.","OneClick module was unable to reinstall successfully."
+"Enable tracking","Enable tracking"
+"Enabling this option will make it easier to track fraudulent payments.","Enabling this option will make it easier to track fraudulent payments."
```

Comment:
Always add a new line (breakline) at the end of file, like that git will not consider the last line as modified just because you create a new line. 
Exemple:
![image](https://user-images.githubusercontent.com/861701/35144896-3a53ff4e-fd06-11e7-98d6-916dcdbc1e5d.png)